### ISSUE TITLE
heroku problem

### ISSUE THREAD

**Issue by rabbykst** [timwis/enviar#96]
Please provide full instruction for installing this repo i'm trying to deploy on heroku but after all done i can't see anything on my page. please help

**Comment by timwis** [comment]
Hi @rabbykst, have you read the instructions in the readme? It talks about setting up a database first - did you do that? What error are you seeing?

**Comment by rabbykst** [comment]
yes i create database & follow the readme but in heroku all are done but when i try to open the app i see there is nothing only a blank page why?

**Comment by timwis** [comment]
Can you send the link to your heroku app? Maybe there's a clue in the javascript console.

**Comment by rabbykst** [comment]
https://sms24.herokuapp.com/ this is my app link please help

**Comment by rabbykst** [comment]
my application log is : 
2018-07-20T19:08:29.490343+00:00 app[web.1]:      date: '2018-07-17T07:49:48.000Z',
2018-07-20T19:08:29.490349+00:00 app[web.1]:      from: '+17174825869',
2018-07-20T19:08:29.490351+00:00 app[web.1]:      to: '+8801812087291',
2018-07-20T19:08:29.490353+00:00 app[web.1]:      body: 'Sent from your Twilio trial account - hi how are you',
2018-07-20T19:08:29.490355+00:00 app[web.1]:      status: 'delivered',
2018-07-20T19:08:29.490356+00:00 app[web.1]:      direction: 'outbound' } }
2018-07-20T19:08:29.870149+00:00 heroku[router]: at=info method=GET path="/" host=sms24.herokuapp.com request_id=9123bac7-e46c-44ac-bcb7-7e6d80f89960 fwd="38.142.118.186" dyno=web.1 connect=1ms service=130ms status=200 bytes=415 protocol=https
2018-07-20T19:08:35.194906+00:00 heroku[router]: at=info method=GET path="/bundle.css" host=sms24.herokuapp.com request_id=387b5326-f7fc-4ee6-b5f5-b5b4cec15d33 fwd="38.142.118.186" dyno=web.1 connect=1ms service=1914ms status=200 bytes=28734 protocol=https
2018-07-20T19:08:37.254754+00:00 heroku[router]: at=info method=GET path="/bundle.js" host=sms24.herokuapp.com request_id=cb918ba9-f170-49be-af48-943fb10644c2 fwd="38.142.118.186" dyno=web.1 connect=1ms service=65ms status=200 bytes=956811 protocol=https
2018-07-20T19:08:42.178224+00:00 heroku[router]: at=info method=GET path="/favicon.ico" host=sms24.herokuapp.com request_id=69d281c6-4064-474b-bc25-6666fa73fc34 fwd="38.142.118.186" dyno=web.1 connect=1ms service=3ms status=200 bytes=415 protocol=https
2018-07-20T19:10:19.154207+00:00 heroku[router]: at=info method=GET path="/" host=sms24.herokuapp.com request_id=001df738-08ac-4411-ac09-63c9f025be0d fwd="38.142.118.186" dyno=web.1 connect=0ms service=7ms status=200 bytes=415 protocol=https
2018-07-20T19:10:20.163333+00:00 heroku[router]: at=info method=GET path="/bundle.css" host=sms24.herokuapp.com request_id=85566c87-4e11-4b10-abb3-728fa7b8059e fwd="38.142.118.186" dyno=web.1 connect=0ms service=5ms status=200 bytes=28734 protocol=https
2018-07-20T19:10:22.260931+00:00 heroku[router]: at=info method=GET path="/bundle.js" host=sms24.herokuapp.com request_id=3fe0c78f-b59f-4773-be89-a9820ee5a8d9 fwd="38.142.118.186" dyno=web.1 connect=0ms service=70ms status=200 bytes=956811 protocol=https
2018-07-20T19:10:27.145991+00:00 heroku[router]: at=info method=GET path="/favicon.ico" host=sms24.herokuapp.com request_id=5833608d-aef0-44b7-a1f6-a54faae9d0ce fwd="38.142.118.186" dyno=web.1 connect=0ms service=3ms status=200 bytes=415 protocol=https

**Comment by timwis** [comment]
From looking at the javascript console on that page, it appears to be a CORS issue: cloudant does not have CORS enabled on it. This should have been setup by enviar's database bootstrapping code though. But it looks like b993926c6fe3080d0c2cbb173c977765792b6a77 renamed a database config file to have a `.tmp` extension. And I honestly can't remember why I would have done that. Perhaps it was done in error. I'm going to create a branch with it renamed back to `.json` -- can you try deploying that one instead and tell me if it works?

EDIT: Go to the [`hotfix/rename-db-config-file` branch](https://github.com/timwis/enviar/tree/hotfix/rename-db-config-file) and click "deploy to heroku" if you don't mind.
